docs/index.html

```diff
@@ -31,7 +31,7 @@
     window.$docsify = {
       name: 'Liatrio\'s DevOps Bootcamp',
       repo: 'liatrio/devops-bootcamp',
-      basePath: 'https://raw.githubusercontent.com/liatrio/devops-bootcamp/master/docs/',
+      // basePath: 'https://raw.githubusercontent.com/liatrio/devops-bootcamp/master/docs/',
```

Comment:
What does this change do? If it was for testing, can we remove the change? If it was an actual intended change, can you remove the code entirely and not leave the comment?

Reply:
Not sure what the issue is when testing but this line will throw a 404 error. Ill remove it.